lib/vimgolf/lib/vimgolf/keylog.rb

```diff
@@ -8,7 +8,9 @@ class Keylog
     def initialize(input, time=Time.now.utc)
       # Force encoding of solution text. Must match string literals.
       # .force_encoding CHANGES THE ORIGINAL STRING!
-      @input = input.force_encoding(Encoding::ASCII_8BIT)
+      # Vim intentionally replaces "<C-C>" (\x03) with "<C-C><C-C>" (Vim #11541).
+      # Convert "<C-C><C-C>" back to "<C-C>" (VimGolf #224).
+      @input = input.force_encoding(Encoding::ASCII_8BIT).gsub("\x03\x03", "\x03")
```

Comment:
I [originally](https://github.com/igrigorik/vimgolf/pull/366/commits/4ae04ee7b1661f4910e30988e349317a221e2c96) had the comment and update on separate lines, after the `input.force_encoding` line, but I moved them both here after a rubocop error from too many lines.